### ISSUE TITLE
508 tool design feedback

### ISSUE THREAD

**Issue by ericronne** [18F/visual-design#38]
Issue #26 itemizes many to dos which stand between a fully *Official* release of this [WIP accessibility tool](https://toolness.github.io/accessible-color-matrix/) (@toolness's side/passion project), but i'm opening this issue to capture general comments, concerns, questions, etc. 

Do you think something like this would be useful to you?
If so, how do you see it fitting into your workflow?
What might you do differently?
What's missing?

Check out the invision prototype of proposed design changes. But please note that as a "side project," we can't guarantee this will progress any further atm :/
Thx!

### [Invision prototype](https://gsa.invisionapp.com/share/2MARWJ2UD#/222604599_default)

**Comment by thecapacity** [comment]
@ericronne I see this as really interesting especially if you could couple it with one of those "contrast scores" that @nickbristow taught me about. I envision the that there might be a row underneath the color boxes (or somewhere along side the hex values) that states the scoring for contrast, and maybe it turns Red/Yellow/Green depending on how close to the right threshold it is, along w/ some sort of "general explanation" 

tl;dr I love the "background" toggle, and think you should put a score on there so someone knows when/if the proper contrast ratios are met by the color selections.

**Comment by amberwreed** [comment]
@ericronne Is this ready to be closed?

**Comment by ericronne** [comment]
I don't know the status of this utility. I think it was sidelined by client work.
cc @toolness

**Comment by toolness** [comment]
I made a submission to a google form to have clearance to work on it as an official 18F project early this year, shortly after the rule whereby we could only work on internal projects that were approved by 18F management was put into place, but I never got a response back. :(

**Comment by jenniferthibault** [comment]
The tool is built but not currently maintained, closing since this isn't active.
